lib/features/manage_account/presentation/forward/forward_controller.dart

```diff
@@ -319,12 +320,12 @@ class ForwardController extends BaseController {
     );
   }
 
-  void handleExceptionCallback(BuildContext context, bool isListEmailEmpty) {
-    if (isListEmailEmpty) {
+  void handleExceptionCallback(BuildContext context, Exception exception) {
+    if (exception is RecipientListIsEmptyException) {
       appToast.showToastErrorMessage(
         context,
         AppLocalizations.of(context).emptyListEmailForward);
-    } else {
+    } else if (exception is RecipientListWithInvalidEmailsException) {
       appToast.showToastErrorMessage(
         context,
         AppLocalizations.of(context).incorrectEmailFormat);
```

Comment:
There's no `else` clause. Should we handle it?

Reply:
No we don't have any other cases yet.